code-style/git/README.md

```diff
@@ -270,16 +275,23 @@ scope: {
 The removed `inject` wasn't generally useful for directives so there should be no code using it.
 ```
 
+## Curating Your Commit History
+
+There are a number of reasons why you may want to "rewrite" the history of your work.
+
+**Before requesting a review of your work**, you may want to clean up your commit history to better "tell the story" of what you did. Using interactive rebasing, you can curate your commit history to combine (squash), reorder, or reword your commits to better onboard the reviewer to your pull request.
```

Comment:
I'd suggest linking to resources that the reader can use, like https://sparkbox.com/foundry/interactive_rebasing_curates_commits_to_speed_up_pull_request_review_process.